src/app/component/CategoryPagination/CategoryPagination.style.scss

```diff
@@ -26,22 +25,22 @@
             content: '';
         }
 
-        @include mobile {
-            &:first-child {
+        &:first-child {
+            @include mobile {
                 padding-left: 0;
             }
         }
 
         &:last-child {
             margin-bottom: 0;
         }
+    }
+}
 
-        a {
-            color: var(--color-pagination-link-text);
+.PaginationLink {
+    color: var(--color-pagination-link-text);
 
-            &.active {
-                color: var(--color-pagination-link-active-text);
-            }
-        }
+    &_active {
+        color: var(--color-pagination-link-active-text);
```

Comment:
You can try setting `--color-pagination-link-text: var(--color-pagination-link-active-text)`, but your solution is also fine.